src/components/Chart/Chart.js

```diff
@@ -1,41 +1,152 @@
-import React from 'react';
-import './Chart.css';
```

Comment:
I think we're gonna use this css file eventually

Reply:
ok cool, will add it back!